### ISSUE TITLE
Event markers tooltip remains disabled even though "enabletooltip" is true.

### ISSUE THREAD

**Issue by ppeyro** [PredixDev/px-vis-timeseries#67]
When you configure event with "enabletooltip": true, the tooltip doesn't trigger on hover over the marker.

This seems specific to the timeseries chart!?! I tried the vis subcomponent https://www.predix-ui.com/#/elements/vis/px-vis/px-vis-event and the tooltip does trigger on hover over the event as expected.  

## Steps to reproduce the problem:

Either use the sample code below, or the dynamic chart generator https://www.predix-ui.com/px-vis-demos/px-vis-demos/dynamic_add.html with event configured (and disable tooltip not checked off, of course) and create a timeseries chart, then hover over the events. 

## Environment:
Browser: Chrome
Timeseries: 3.2.1
vis: 4.3.3

### px-vis-event subcomponent on hover:

![screen shot 2017-10-31 at 10 14 47 am](https://user-images.githubusercontent.com/10835611/32252734-144bb102-be54-11e7-9937-dafb19a02835.png)

### Timeseries with enabletooltip for event set to true

![screen shot 2017-10-31 at 3 55 39 pm](https://user-images.githubusercontent.com/10835611/32252740-170bed30-be54-11e7-9e47-873004566afd.png)

## Code example: 

```<px-vis-timeseries
          id="timeseries1"
          selection-type="xy"
          render-to-canvas
          event-config= '{
            "Default":{
              "color": "blue",
              "icon":"px-fea:home",
              "type": "fa",
              "offset":[0,0],
              "enableTooltip": true
            },
            "Recalibrate":{
              "color": "blue",
              "icon":"px-fea:deployments",
              "type": "fa",
              "offset":[0,0],
              "enableTooltip": true
            },
            "Fan start":{
              "color": "red",
              "icon": "px-fea:alerts",
              "type": "fa",
              "offset":[1,0],
              "enableTooltip": true
            },
            "Fan stop":{
              "color": "purple",
              "icon": "px-fea:bug",
              "type": "fa",
              "offset":[0,1],
              "size":"20",
              "enableTooltip": true
            }
          }'

**Comment by benoitjchevalier** [comment]
Suspect the marker canvas goes on top of the events svg, preventing the tooltip interaction

**Comment by benoitjchevalier** [comment]
fixed in vis 4.41
